internal/event/movie_created.go

```diff
@@ -0,0 +1,35 @@
+package event
+
+import (
+	"time"
+
+	"github.com/maestre3d/academy-go-q12021/internal/valueobject"
+)
+
+// MovieCreated a movie was created
+type MovieCreated struct {
+	ID          string    `json:"movie_id"`
+	DisplayName string    `json:"display_name"`
+	Directors   []string  `json:"director"`
+	ReleaseYear int       `json:"release_year"`
+	CreateTime  time.Time `json:"create_time"`
+}
+
+func NewMovieCreated(id valueobject.MovieID, name valueobject.DisplayName,
+	year valueobject.ReleaseYear, directors ...valueobject.DisplayName) MovieCreated {
+	return MovieCreated{
+		ID:          string(id),
+		DisplayName: string(name),
+		Directors:   valueobject.MarshalDirectorsPrimitive(directors...),
+		ReleaseYear: int(year),
+		CreateTime:  time.Now().UTC(),
+	}
+}
+
+func (c MovieCreated) Kind() string {
```

Comment:
All the exported functions, structs, interfaces, variables, etc, should have description.
Do it this for all of them